truss/config/trt_llm.py

```diff
@@ -3,8 +3,9 @@
 from enum import Enum
 from typing import Optional
 
-from pydantic import BaseModel, field_validator
+from pydantic import BaseModel, field_validator, model_validator
```

Comment:
These seem to be pydantic V2 dependencies? I think there were issues in the past in dropping V1 and moving to V2.
@bdubayah

Reply:
@dsingal0

Reply:
@marius-baseten the model_validator import should be removed and the validation should be left as it was before (in the constructor) right?

Reply:
It seems `field_validator` is also a v2 API. Can we move this discussion to truss slack? I would really like to see us moving to v2 and ditch v1, but a while ago we had big issues with a customers because of that... let's check what the current state is